Source/Logger/trace.cpp

```diff
@@ -14,42 +14,6 @@
 namespace
 {
 
-//------------------------------------------------------------------------------
-// CHAR overloads for printf style functions used in tracing
-//------------------------------------------------------------------------------
-#if HC_CHAR_IS_WIDE
-
-template<size_t SIZE>
-int stprintf_s(wCHAR(&buffer)[SIZE], _Printf_format_string_ wCHAR const* format, ...)
-{
-    va_list varArgs = nullptr;
-    va_start(varArgs, format);
-    auto result = vswprintf_s(buffer, format, varArgs);
-    va_end(varArgs);
-    return result;
-}
-
-int stprintf_s(wCHAR* buffer, size_t size, _Printf_format_string_ wCHAR const* format, ...)
-{
-    va_list varArgs = nullptr;
-    va_start(varArgs, format);
-    auto result = vswprintf_s(buffer, size, format, varArgs);
-    va_end(varArgs);
-    return result;
-}
-
-template<size_t SIZE>
-int vstprintf_s(wCHAR(&buffer)[SIZE], _Printf_format_string_ wCHAR const* format, va_list varArgs)
-{
-    return vswprintf_s(buffer, format, varArgs);
-}
-
-void OutputDebugStringT(wCHAR const* string)
-{
-    OutputDebugStringW(string);
-}
-
-#else
 
 template<size_t SIZE>
 int stprintf_s(char(&buffer)[SIZE], _Printf_format_string_ char const* format ...)
```

Comment:
Now that my crazy overloads are gone, we need to change the code that calls to them

Reply:
Feel free to fix in a new PR